packages/x-date-pickers/src/StaticDatePicker/StaticDatePicker.test.tsx

```diff
@@ -67,4 +67,22 @@ describe('<StaticDatePicker />', () => {
     expect(getYearButton(2030)).to.have.attribute('disabled');
     expect(getYearButton(2031)).not.to.have.attribute('disabled');
   });
+
+  it('prop `disabled` – disables all days', () => {
```

Comment:
In other tests, they do the same check for `readOnly`, I did not look into those for now but maybe it also make sense here.